src/fe-gtk/gtkutil.c

```diff
@@ -598,6 +599,10 @@ gtkutil_window_new (char *title, char *role, int width, int height, int flags)
 		gtk_window_set_transient_for (GTK_WINDOW (win), GTK_WINDOW (parent_window));
 		gtk_window_set_destroy_with_parent (GTK_WINDOW (win), TRUE);
 	}
+	if (flags & START_ICONIFIED)
```

Comment:
Any idea on how am I supposed to fix that? I wouldn't continue until you give me a pointer to the solution that you'd like to have.

Also, please try to give me all the feedback in batch. It's more time-efficient for us.